base/server/python/pki/server/cli/acme.py

```diff
@@ -1018,8 +1068,26 @@ def execute(self, argv):
         issuer_conf = os.path.join(acme_conf_dir, 'issuer.conf')
         config = {}
 
-        logger.info('Loading %s', issuer_conf)
-        pki.util.load_properties(issuer_conf, config)
+        if issuer_type:
+            source = '/usr/share/pki/acme/issuer/%s/issuer.conf' % issuer_type
```

Comment:
`%` is a printf style formatter used in python2. Since we are targetting python3+, we should try using `.format()`

Ref: https://stackoverflow.com/q/12382719

Reply:
Fixed.